kafka_consumer/datadog_checks/kafka_consumer/legacy_0_10_2.py

```diff
@@ -469,7 +462,7 @@ def _send_event(self, title, text, tags, event_type, aggregation_key, severity='
         """Emit an event to the Datadog Event Stream."""
         event_dict = {
             'timestamp': int(time()),
-            'source_type_name': self.SOURCE_TYPE_NAME,
+            'source_type_name': self.__NAMESPACE__,
```

Comment:
No need since https://github.com/DataDog/integrations-core/blob/213e392f004e8fa8590dfebdef4fda6f142b0730/datadog_checks_base/datadog_checks/base/checks/base.py#L592-L593